### PR TITLE
LPS-74425

### PR DESCRIPTION
**Pull request by mbowerman**

### CODE REVIEW

File: portal-impl/src/com/liferay/portal/upgrade/v7_0_0/UpgradeResourcePermission.java [hunk 1]
```diff
@@ -57,76 +57,97 @@ protected void upgradeResourcePermissions() throws Exception {
...
-		}
+		String inClause = sb.toString();
+
+		_updatePrimKeyIds(
+			"update ResourcePermission set primKeyId = 0 where name = ? and " +
+				"(primKey like '%_LAYOUT_%' or primKey " + inClause + ")",
+			name, primKeys);
+
+		_updatePrimKeyIds(
+			"update ResourcePermission set primKeyId = CAST_LONG(primKey" +
```
Comment (Preston-Crary) [inline portal-impl/src/com/liferay/portal/upgrade/v7_0_0/UpgradeResourcePermission.java]: Looks like you missed the and name = ?

Comment (Preston-Crary) [reply portal-impl/src/com/liferay/portal/upgrade/v7_0_0/UpgradeResourcePermission.java]: Nevermind, just not reading it right sorry.

Comment (Preston-Crary) [reply portal-impl/src/com/liferay/portal/upgrade/v7_0_0/UpgradeResourcePermission.java]: You have name = ? in the above query but not in this one....

Comment (mbowerman) [reply portal-impl/src/com/liferay/portal/upgrade/v7_0_0/UpgradeResourcePermission.java]: Fixed it

**Comment by Preston-Crary** [comment]
ci:stop

**Comment by Preston-Crary** [comment]
ci:stop

**Comment by Preston-Crary** [comment]
I'll retest after the CI comes back with the response.

**Comment by Preston-Crary** [comment]
CI still hasn't come back... can you resend?

**Comment by mbowerman** [comment]
Resent at https://github.com/Preston-Crary/liferay-portal/pull/391

**Pull request closed without merging**